.circleci/config.yml

```diff
@@ -248,7 +248,7 @@ workflows:
             - build_and_test
           filters:
             branches:
-              only: release/*
+              only: /^release.*/
```

Comment:
So this matches `release` and `release-1`, but rejects `1-release` and `ars-release` — does that summarize the change well?

Reply:
yea, it is looking for branches starting with "release"